src/middleware/github-webhook-middleware.ts

```diff
@@ -203,6 +203,11 @@ export const GithubWebhookMiddleware = (
 				continue;
 			}
 
+			if (await isBlocked(jiraHost, gitHubInstallationId, context.log)) {
+				context.log.info({ jiraHost, gitHubInstallationId }, `Block webhook processing`);
+				continue;
```

Comment:
Why are we continuing here? If we want to block the webhook from processing shouldn't we return?